wg-serving/charter.md

```diff
@@ -0,0 +1,87 @@
+  from practitioners, distros, and vendors. Provide concrete input to other SIGs
+  and WGs around needs for identified requirements. Do it in partnership
+  with existing ecosystem projects like kServe, Seldon, Kaito, and
+  others to identify, extract, or implement common shared problems (like Kueue
+  abstracted deferred scheduling for multiple batch frameworks).
+- Specific areas of improvement include:
+  - Directly improve key kubernetes workload controllers when used with
+    accelerators and the most common inference serving frameworks and model
+    servers.
+  - Explore new projects that improve orchestration, scaling, and load balancing
+    of inference workloads and compose well with other workloads on Kubernetes
+  - Being able to run serving workloads safely while giving up
+    available slack capacity to batch frameworks
+
+### Out of scope
+
+- Training and batch inference, which are covered by WG Batch.
+- Addition of new API kinds that serve a specific models. The focus should be on
+  general APIs that frameworks can build on top of.
+- Ability to describe the workflows for serving workloads is out of scope,
+  Kubernetes will offer building blocks to MLOps platforms to build those.
+
+## Stakeholders
+
+Stakeholders in this working group span multiple SIGs that own parts of the
+code in core kubernetes components and addons.
+
+- SIG Apps as an primary SIG
+- SIG Architecture
+- SIG Node
+- SIG Scheduling
+- SIG Autoscaling
+- SIG Network
+- SIG Instrumentation
+- SIG Storage
+
+## Deliverables
+
+The list of deliverables include the following high level features:
+
+- To SIG Apps:
+  - Ability to express the model serving workloads with easy to understand logical
+    objects with the ability to scale to multi-host 
+- To SIG Scheduling and Autoscaling
+  - Faster scaling up and down
+  - Ability to preempt workloads
+- To SIG Node:
+  - Runtime support for Pods preemption
+  - Runtime support for devices partitioning
+
+## Roles and Organization Management
+
+This wg adheres to the Roles and Organization Management outlined in [wg-governance]
+and opts-in to updates and modifications to [wg-governance].
+
+[wg-governance]: /committee-steering/governance/wg-governance.md
+
+Additionally, the wg commits to:
+
+- maintain a solid communication line between the Kubernetes groups and the wider CNCF community;
+- submit a proposal to the KubeCon/CloudNativeCon maintainers track;
+
+## Timelines and Disbanding
+
+As a first mandate, the wg will define a roadmap in the first quarter of operation.
+We believe there will be a set of features the Working Group can identify and deliver
+that will enable the majority of frameworks operate natively on Kubernetes.
+
+There is no expectations that the Working Group will be converted into SIG long term,
+however, there is a chance that a separate project or a sizeable sub-component of SIG Apps can be 
+created as a result of a Working Group.
```

Comment:
I'd like to see clearer exit criteria, maybe something along the deliverables? Iow. after achieving them and the items identified in the roadmap?

Reply:
I was generally following https://github.com/kubernetes/community/blob/master/wg-batch/charter.md#timelines-and-disbanding.

We don't yet know exact deliverables and as it's said in text we may end up developing a new sub project or it may be an external plugin. It is something we will need to decide.

What specific points are you looking for in this text?

Reply:
Let's try to sketch at least the general exit criteria, for example try adding something like that to the first paragraph:

```
Achieving the aforementioned deliverables, also mentioned in the `In Scope` section,
will allow us to decide when to disband this WG. 
```

Reply:
+1 to @soltysh 's comment. We've had several WGs that have gone on for years making minimal progress with no real exit criteria >_< Even if its fairly plane like that, its still worth it to have written down.

Reply:
Yes, this has been a repeat problem, workgroups are supposed to be scoped and finite, let's try to outline what the exit criteria are now.

Reply:
added, makes sense. Thank you for clarifying concerns